xla/service/gpu/triton_fusion_analysis_test.cc

```diff
@@ -582,6 +582,42 @@ ENTRY e {
                                     /*subfragments=*/ElementsAre(4))));
 }
 
+TEST_F(TritonDotAnalysisTest, BroadcastFromTriviallySizedDimensionIsSupported) {
```

Comment:
Do we have negative tests for the broadcasts that should be filtered out? It's a bit hard to gauge whether the code is as tight as it should be.

Reply:
Added BroadcastWithinDimensionIsNotSupported.